code/datums/status_effects/status_effect.dm

```diff
@@ -51,7 +51,7 @@
 	if(!owner)
 		qdel(src)
 		return
-	if(tick_interval < world.time)
+	if(tick_interval <= world.time)
```

Comment:
this was changed as `transient` effects tick every 0.2 seconds. so the tick interval was the same as the world time, so it only fired half of the time.